pages/common/duplicacy.md

```diff
@@ -0,0 +1,36 @@
+# duplicacy
+
+> A lock-free deduplication cloud backup tool.
+> More information: <https://github.com/gilbertchen/duplicacy/wiki>.
```

Comment:
The wiki lists all subcommonds and documentation, but not an overall description. I'd use the README for that
```suggestion
> More information: <https://github.com/gilbertchen/duplicacy>.
```